content/docs/fairing/gcp-local-notebook.md

```diff
@@ -0,0 +1,228 @@
++++
+title = "Train and deploy on GCP from a local notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Google Cloud Platform from a local notebook."
+weight = 30
++++
+
+This guide introduces you to using Kubeflow Fairing to train and deploy a
+model to Kubeflow on Google Kubernetes Engine, and Google Cloud ML Engine.
+As an example, this guide uses a local notebook to demonstrate how to:
+
+*  Train an XGBoost model in a local notebook,
+*  Use Kubeflow Fairing to train an XGBoost model remotely on Kubeflow,
+*  Use Kubeflow Fairing to train an XGBoost model remotely on Google Cloud ML Engine, 
+*  Use Kubeflow Fairing to deploy a trained model to Kubeflow, and
+*  Call the deployed endpoint for predictions.
+
+This guide has been tested on Linux and Mac OS X. Currently, this guide has not been
+tested on Windows.
+
+## Clone the Kubeflow Fairing Repository
```

Comment:
s/Repository/repository

(We use sentence case for headings within a doc.)

Reply:
Thanks for pointing that out, changed.